.travis.yml

```diff
@@ -13,19 +13,14 @@ sudo: required
 services:
   - docker
 
-# Install dependencies for launching minikube cluster.
-before_install:
-  - sudo apt-get update
-  - sudo apt-get install -y socat ebtables
-
 # Install must be set to prevent default `go get` to run.
 # The dependencies have already been vendored by `dep` so
 # we don't need to fetch them.
 install:
   -
 
 script:
-  - bash -x ./scripts/pre-commit.sh
+  - bash -c "DEBUG=y CONFIGURE_INSECURE_REGISTRY=y ./scripts/pre-commit.sh"
```

Comment:
(No action required) If the only point of `DEBUG` is to enable `-x`, can it be discontinued in favor of calling the script with `-x`?

Reply:
Yep, agreed. I had opted for this as I was having difficulty passing a variable and invoking with `-x` because when using `-x` the next argument must be an executable bash file. I've addressed that now.